test/OpenTelemetry.Instrumentation.AspNetCore.Tests/RouteTests/TestApplication/TestApplicationFactory.cs

```diff
@@ -154,4 +162,38 @@ private static WebApplication CreateRazorPagesApplication()
 
         return app;
     }
+
+    private static WebApplication CreateExceptionHandlerApplication()
+    {
+        var builder = WebApplication.CreateBuilder();
+        builder.Logging.ClearProviders();
+
+        var app = builder.Build();
+
+        app.UseExceptionHandler(exceptionHandlerApp =>
+        {
+            exceptionHandlerApp.Run(async context =>
+            {
+                context.Response.StatusCode = StatusCodes.Status500InternalServerError;
+                var exceptionHandlerPathFeature = context.Features.Get<IExceptionHandlerPathFeature>();
+                await context.Response.WriteAsync(exceptionHandlerPathFeature?.Error.Message ?? "An exception was thrown.");
+            });
+        });
+
+        app.Urls.Clear();
+        app.Urls.Add("http://[::1]:0");
+
+// TODO: Remove this condition once ASP.NET Core 8.0.2.
+// Currently, .NET 8 has a different behavior than .NET 6 and 7.
+// This is because ASP.NET Core 8+ has native metric instrumentation.
+// When ASP.NET Core 8.0.2 is released then its behavior will align with .NET 6/7.
+// See: https://github.com/dotnet/aspnetcore/issues/52648#issuecomment-1853432776
```

Comment:
Adding spaces to make dotnet format happy

```suggestion
        // TODO: Remove this condition once ASP.NET Core 8.0.2.
        // Currently, .NET 8 has a different behavior than .NET 6 and 7.
        // This is because ASP.NET Core 8+ has native metric instrumentation.
        // When ASP.NET Core 8.0.2 is released then its behavior will align with .NET 6/7.
        // See: https://github.com/dotnet/aspnetcore/issues/52648#issuecomment-1853432776
```